src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployer.java

```diff
@@ -275,10 +275,14 @@ protected void createStatefulSet(AppDeploymentRequest request) {
 
 		String storageClassName = this.deploymentPropertiesResolver.getStatefulSetStorageClassName(kubernetesDeployerProperties);
 
+		String volumeClaimTemplateName = this.deploymentPropertiesResolver.getStatefulSetVolumeClaimTemplateName(kubernetesDeployerProperties);
+
+		volumeClaimTemplateName = StringUtils.hasText(volumeClaimTemplateName) ? volumeClaimTemplateName : appId;
```

Comment:
If not mistaken the `appId`s are used in by the app runtime information returned to the clients and perhaps by skipper. 
Is there a risk for the above change to affect any existing skipper behavior?

Reply:
The `appId` is still the default and it's importance as the name of the volume  here is just being unique for the app being deployed. Otherwise, we provide the option to make the volume name configurable.

Reply:
got it